_posts/2017-08-25-aug-25.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "This Week In Erlang Aug 25"
+description: ""
+date: 2017-08-25
+tags: [erlang, news]
+comments: false
+share: true
+---
+
+Hello and welcome to "This week in Erlang"! As expected, there was a lot of action on the OTP side this week, and maybe a little too slow everywhere else. However, there was a piece of great news that came out this week. Read on!
+
+### Articles and Blog posts
+ * Bet365's CEO announced that they will be buying all of Basho's remaining assets and making everything open source: <http://lists.basho.com/pipermail/riak-users_lists.basho.com/2017-August/019500.html>
```

Comment:
I found 5 misspelled words in this sentence: 
- <http
- Basho
	- *Suggested Replacements:* basso, basto, bash, baho
- CEO
	- *Suggested Replacements:* cro, eco, leo, cee, cwo, eo, cero, coe, coo, neo, cto, heo, cep, cpo, yeo, ce, clo, co, cen, geo, cho
- Bet365
- *
	- *Suggested Replacements:* , k, p, u, e, z, j, o, t, d, y, i, n, s, c, x, h, m, r, b, w, g, l, q, a, v, f